include/gatekeeper_lls.h

```diff
@@ -361,13 +361,27 @@ struct nd_opt_lladdr {
 #define ND_NEIGH_PKT_LLADDR_MIN_LEN(l2_len) (ND_NEIGH_PKT_MIN_LEN(l2_len) + \
 	sizeof(struct nd_opt_lladdr))
 
+/* Minimum size of an IPv4 ping packet. */
+#define ICMP_PKT_MIN_LEN(l2_len) (l2_len + sizeof(struct ipv4_hdr) + 8)
```

Comment:
What are the 8 extra bytes? If you can't make it into an expression, write a comment explaining it.